lonestar/analytics/cpu/kcore/kcore.cpp

```diff
@@ -232,7 +229,10 @@ void kCoreSanity(Graph& graph) {
 
 int main(int argc, char** argv) {
   galois::SharedMemSys G;
-  LonestarStart(argc, argv, name, desc, url);
+  LonestarStart(argc, argv, name, desc, nullptr, inputFile.c_str());
+
+  galois::StatTimer totalTime("TimerTotal");
+  totalTime.start();
```

Comment:
Missing stop.

Going to stop marking them from this point onward because I'm 99% percent sure this is on purpose at this point (I just checked the destructor and it does indeed call stop).

One thing that I do want to comment about though is that this may be confusing to a code reader who isn't aware that the destructor for a StatTimer calls stop. One could argue that it's something obvious, but leaving things up to assumption doesn't sit well with me.

For the record, I'm fine with the way it currently is too; I simply prefer to be as explicit as possible in code to avoid confusion to new code readers.